test/lib/smart-survey-api.test.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const sinon = require('sinon');
+const chai = require('chai').use(require('sinon-chai'));
+chai.should();
+const proxyquire = require('proxyquire');
+let stubAxios = {};
+
+// Proxyquire syntax: first argument: point to your module (which holds your dependency)
+// 2nd argument: An object with the key: module name to proxy & the value the empty object above
+// So this means smartSurveyAxiosAPI will now point to this empty object, i.e. a stub
+const smartSurveyAxiosAPI = proxyquire('../../lib/smart-survey-api', {
+  'axios': stubAxios,
+});
+
+// create an empty function using not using arrow functions because using context
+var stubSmartSurvey = function empty() {
```

Comment:
why are you naming the function different to what it's assigned to?
this will be executable with both `stubSmartSurvey()` and `empty()` which is probably not what you intended